app/src/main/java/com/codesses/priceconverter/data/repo/PriceConverterRepositoryImpl.kt

```diff
@@ -0,0 +1,39 @@
+package com.codesses.priceconverter.data.repo
+
+import com.codesses.priceconverter.data.remote.ApiInterface
+import com.codesses.priceconverter.data.remote.dto.BitPayExchangeRatesResponse
+import com.codesses.priceconverter.domain.repo.PriceConverterRepository
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.callbackFlow
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import javax.inject.Inject
+
+class PriceConverterRepositoryImpl @Inject constructor(
+    private val api: ApiInterface
+) : PriceConverterRepository {
+    //
+//    override suspend fun getExchangeRates(): Flow<BitPayExchangeRatesResponse> = callbackFlow {
```

Comment:
Probably will be deleted before merge, but just to keep track of it, lets not keep commented code, delete it instead.